### PR TITLE
using the is pattern expression

### PR DESCRIPTION
**Pull request by SergerGood**

### CODE REVIEW

File: RepoDb.Core/RepoDb/QueryField/ParseExpression.cs [hunk 1]
```diff
@@ -443,7 +431,7 @@ internal static ClassProperty GetProperty<TEntity>(Expression expression)
         /// <returns></returns>
         internal static ClassProperty GetProperty<TEntity>(LambdaExpression expression)
             where TEntity : class =>
-            GetProperty<TEntity>(expression.ToLambda().Body);
+            GetProperty<TEntity>(expression.Body);
```
Comment (mikependon) [inline RepoDb.Core/RepoDb/QueryField/ParseExpression.cs:434]: Is this working?

File: RepoDb.Core/RepoDb/QueryGroup/ParseExpression.cs [hunk 2]
```diff
@@ -279,7 +252,7 @@ private static QueryGroup Parse<TEntity>(MethodCallExpression expression)
         /// <param name="expression"></param>
         /// <returns></returns>
         internal static ExpressionType? GetNodeType(LambdaExpression expression) =>
-            GetNodeType(expression.ToLambda().Body);
+            GetNodeType(expression.Body);
```
Comment (mikependon) [inline RepoDb.Core/RepoDb/QueryGroup/ParseExpression.cs:255]: Also this?

File: RepoDb.Core/RepoDb/QueryField/ParseExpression.cs (hunk 1 above)
Comment (SergerGood) [reply RepoDb.Core/RepoDb/QueryField/ParseExpression.cs:434]: Yes, It was same cast. LambdaExpression was cast to LambdaExpression.

File: RepoDb.Core/RepoDb/QueryGroup/ParseExpression.cs (hunk 2 above)
Comment (SergerGood) [reply RepoDb.Core/RepoDb/QueryGroup/ParseExpression.cs:255]: And also this.

**Pull request merged**